tests/test_mlip_calculators.py

```diff
@@ -123,16 +128,32 @@ def test_extra_mlips(architecture, device, kwargs):
     "kwargs",
     [
         {
-            "model_path": MODEL_PATH / "v5.27.2024/best_model.pt",
-            "model": MODEL_PATH / "v5.27.2024/best_model.pt",
+            "architecture": "alignn",
+            "model_path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+            "model": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+        },
+        {
+            "architecture": "alignn",
+            "model_path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+            "path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+        },
+        {
+            "architecture": "sevennet",
+            "model_path": SEVENNET_PATH,
+            "path": SEVENNET_PATH,
+        },
+        {
+            "architecture": "sevennet",
+            "model_path": SEVENNET_PATH,
+            "model": SEVENNET_PATH,
         },
         {
-            "model_path": MODEL_PATH / "v5.27.2024/best_model.pt",
-            "path": MODEL_PATH / "v5.27.2024/best_model.pt",
+            "architecture": "sevennet",
+            "model_path": "",
         },
     ],
 )
-def test_extra_mlips_invalid(kwargs):
+def test_alignff_invalid(kwargs):
```

Comment:
```suggestion
def test_extra_mlips_invalid(kwargs):
```
I think this got lost in a rebase?